include/itkANTSRegistration.h

```diff
@@ -121,6 +127,14 @@ class ANTSRegistration : public ProcessObject
   itkSetStringMacro(TypeOfTransform);
   itkGetStringMacro(TypeOfTransform);
 
+  /** The metric for the affine part (GC, mattes, meansquares). */
```

Comment:
Please enumerate the possible values in the docstring

Reply:
Waiting on resolution of this: https://github.com/ANTsX/ANTs/issues/1668